app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -66,11 +72,18 @@
 	public RecyclerView mRecyclerView;
 	public RelativeLayout splash;
 
+	public static ApolloClient apolloClient;
+	public static final boolean JSON_FALLBACK = false;
+
 	@Override
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setTitle("Eatery");
 		setContentView(R.layout.activity_main);
+
+		//Preload Eateries
+        NetworkUtilities.getEateries();
```

Comment:
this isn't formatted properly, can you check your indentation settings & make sure they sync with what @ningning621 has?